lib/engine/game/base.rb

```diff
@@ -577,19 +577,22 @@ def connect_hexes
       end
 
       def next_round!
+        if (game_end = game_ending?)
```

Comment:
can you do if (_reason, after = game_ending?)

Reply:
yep, forgot you can do that in ruby :)